src/components/portrait/portrait.test.js

```diff
@@ -0,0 +1,281 @@
+import React from "react";
+import Portrait from "./portrait.component";
+import Box from "../box";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import { icon, getDataElementByValue } from "../../../cypress/locators";
+
+import {
+  portraitPreview,
+  portraitInitials,
+  portraitAlt,
+} from "../../../cypress/locators/portrait";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const testImage = [
+  "https://www.gravatar.com/avatar/cceff1ad774bf89b1f75cb6e5005333c?s=40&d=404",
+];
+
+const PortraitComponent = ({ ...props }) => {
+  return <Portrait {...props} />;
+};
+
+const TooltipPortraitComponent = ({ ...props }) => {
+  return (
+    <Portrait
+      tooltipMessage="Rebecca Smith"
+      tooltipIsVisible
+      src="https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light"
+      {...props}
+    />
+  );
+};
+
+context("Tests for Portrait component", () => {
+  describe("check props for Portrait component", () => {
+    it.each([
+      [
+        "chris.barber@sage.com",
+        "https://www.gravatar.com/avatar/cceff1ad774bf89b1f75cb6e5005333c?s=40&d=404",
+      ],
+    ])("should check gravatar for Portrait components", (emailID, srcImage) => {
+      CypressMountWithProviders(<PortraitComponent gravatar={emailID} />);
+      portraitPreview()
+        .should("have.css", "height", "40px")
+        .and("have.css", "width", "40px");
+      portraitPreview().children().should("have.attr", "src", srcImage);
+    });
+
+    it.each([
+      "https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light",
+
+      "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404",
+    ])("should check custom image url for Portrait component ", (srcImage) => {
+      CypressMountWithProviders(<PortraitComponent src={srcImage} />);
+      portraitPreview()
+        .should("have.css", "height", "40px")
+        .and("have.css", "width", "40px");
+      portraitPreview().children().should("have.attr", "src", srcImage);
+    });
+
+    it.each([
+      ["XS", 24],
+      ["S", 32],
+      ["M", 40],
+      ["ML", 56],
+      ["L", 72],
+      ["XL", 104],
+      ["XXL", 128],
+    ])(
+      "should check %s size for Portrait component ",
+      (size, heightAndwidth) => {
+        CypressMountWithProviders(<PortraitComponent size={size} />);
+        portraitPreview()
+          .should("have.css", "height", `${heightAndwidth}px`)
+          .and("have.css", "width", `${heightAndwidth}px`);
```

Comment:
Variable should be in camel case, like `heightAndWidth` rather than `heightAndwidth`.